modules/repo_sync/src/Controller/RepoSyncViewController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\devportal_repo_sync\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\devportal_repo_sync\Exception\DevportalRepoSyncConnectionException;
+use Drupal\devportal_repo_sync\Service\Client;
+
+/**
+ * Class RepoSyncViewController.
+ */
+class RepoSyncViewController extends ControllerBase {
+
+  /**
+   * Hello.
+   *
+   * @param string $uuid
+   *   The UUID of the imported repository.
+   *
+   * @return array
+   *   Return content render array.
+   *
+   * @throws \Exception
+   */
+  public function content($uuid) {
+    $config = $this->config('devportal_repo_sync.config');
+    $client = new Client($config->get('uuid'), hex2bin($config->get('secret')), $config->get('service'));
+
+    try {
+      $result = $client("GET", "/api/import/$uuid", NULL);
+      $result = json_decode(array_pop($result), TRUE);
+
+      $rows = [];
+      foreach ($result as $key => $value) {
+        $rows[] = [$key, $value];
+      }
+    }
+    catch (DevportalRepoSyncConnectionException $e) {
+      $this->messenger()->addError($e->getMessage());
+    }
+
+    $build = [
```

Comment:
This should go inside the `try {}` block.

There should be some kind of an error page in the catch block. Maybe a standard one?